caas/kubernetes/provider/k8s.go

```diff
@@ -72,7 +72,8 @@ const (
 
 	gpuAffinityNodeSelectorKey = "gpu"
 
-	jujudToolDir = "/var/lib/juju/tools"
+	jujuHome     = "/var/lib/juju"
```

Comment:
Should be jujuDataDir
Also, it should be paths.DataDir("kubernetes")

Reply:
changed